src/main/java/uk/gov/ons/census/fwmt/jobservice/service/JobService.java

```diff
@@ -66,37 +64,32 @@ public void processCreate(FwmtActionInstruction rmRequest, Instant messageReceiv
     ProcessorKey key = ProcessorKey.buildKey(rmRequest);
     List<InboundProcessor<FwmtActionInstruction>> processors = createProcessorMap.get(key);
 
-    if (processors == null)
-      processors = Collections.emptyList();
-    else
-      processors = createProcessorMap.get(key).stream().filter(p -> p.isValid(rmRequest, cache)).collect(Collectors.toList());
+    if (processors != null) processors = createProcessorMap.get(key).stream().filter(p -> p.isValid(rmRequest, cache)).collect(Collectors.toList());
 
-    if (processors.size() == 0) {
-      // TODO throw routing error & exit;
+    if (processors != null && processors.isEmpty()) {
       eventManager.triggerErrorEvent(this.getClass(), "Could not find a CREATE processor for request from RM", String.valueOf(rmRequest.getCaseId()), ROUTING_FAILED,
```

Comment:
the method that looks up the map from  the  key im assuming is null at this point since it failed the look up and then never filtered.
this means i can enter fail state with processors being null and empty